src/main/java/com/minecolonies/colony/permissions/Permissions.java

```diff
@@ -3,6 +3,7 @@
 import com.minecolonies.colony.Colony;
 import com.minecolonies.network.PacketUtils;
 import com.minecolonies.util.AchievementUtils;
+import com.minecolonies.util.Log;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.minecolonies.util.Log'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUselessImportCheck)